docs/userguides/platform.md

```diff
@@ -20,6 +20,23 @@ The Platform UI will let you create and manage Clusters using a graphical experi
 The CLI experience is for those working locally who don't want to visit the website, or are locally developing their applications.
 ```
 
+Once you have created your Cluster, you have to fund it so it is made available for your use.
+To do that, use the [`silverback cluster pay create`][silverback-cluster-pay-create] command to fund your newly created cluster.
```

Comment:
suggestion(non-blocking): I wonder if it's worth mentioning that you need a funded Ape account for this to work. Might confuse things maybe, but I'm not sure if all Silverback users will have an Ape environment ready to go.

Reply:
Good suggestion!

Yes, the UI will be more hand-holdy through this experience, only so much I can do via CLI without assuming a certain level of experience from the user